worldedit_commands/init.lua

```diff
@@ -45,14 +45,14 @@ local function chatcommand_handler(cmd_name, name, param)
 	if def.nodes_needed then
 		local count = def.nodes_needed(name, unpack(parsed))
 		safe_region(name, count, function()
-			local success, msg = def.func(name, unpack(parsed))
+			local msg = def.func(name, unpack(parsed))

```

Comment:
Incorrect.
Just disable the `unused_secondaries` warning if it doesn't report anything useful.

Reply:
Checking `worldedit_commands\init.lua`              4 warnings

    worldedit_commands\init.lua:48:10: unused variable 'success'
    worldedit_commands\init.lua:48:10: shadowing upvalue 'success' on line 39
    worldedit_commands\init.lua:55:9: unused variable 'success'
    worldedit_commands\init.lua:55:9: shadowing definition of variable 'success' on line 39

Reply:
You have to replace `success` with `_`.